### ISSUE TITLE
Hi there, is bert-large-uncased-whole-word-masking-finetuned-squad trained for Squad 1.0 or 2.0?

### ISSUE THREAD

**Issue by JianLiu91** [huggingface/transformers#1221]
I mean, whether the training data contains examples with no answer?

**Comment by LysandreJik** [comment]
Hi! I believe this checkpoint originates from the training specified [there](https://huggingface.co/pytorch-transformers/examples.html#squad).

The SQuAD version would then be 1.1!
